src/services/userService.js

```diff
@@ -214,6 +215,37 @@ async function returnAllFavouriteEvents(userId) {
     }
 }
 
+async function approvedUser(userId, requestBody) {
+    const existingUser = await User.findById(userId);
+    if (!existingUser) {
+        throw new Error('User not found');
+    }
+    existingUser.isApproved = requestBody.isApproved;
+
+    const newRecord = await existingUser.save();
+    return createToken(newRecord);
+}
+
+async function organizersAwaitingForApproval() {
+    const allUsers = await User.find({ isApproved: false });
```

Comment:
Isn't this returning all users, not only organizers ?

Reply:
fixed